rttr-de.po

```diff
@@ -3367,11 +3380,10 @@ msgid ""
 msgstr ""
 "Du kannst Goldadern vollständig entfernen oder in Eisen, Kohle oder Granit umwandeln\n"
 "\n"
-"Es ist sinnvoll Gold durch Eisen zu ersetzen, da die überschüssige Kohle, auf den meisten Karten,\n"
+"Es ist sinnvoll Gold durch Eisen zu ersetzen, da die überschüssige Kohle auf den meisten Karten\n"
 "dann für die Herstellung von Waffen verwendet werden kann."
```

Comment:
"... da dann die überschüssige Kohle ..." ? und das "dann" in der zweiten Zeile raus?